server/vcs/provider/github/team_fetcher.go

```diff
@@ -0,0 +1,36 @@
+package github
+
+import (
+	"context"
+	gh "github.com/google/go-github/v45/github"
+	"github.com/palantir/go-githubapp/githubapp"
+	"github.com/pkg/errors"
+)
+
+type TeamMemberFetcher struct {
+	ClientCreator githubapp.ClientCreator
+	Org           string
+}
+
+func (t *TeamMemberFetcher) ListTeamMembers(ctx context.Context, installationToken int64, teamSlug string) ([]string, error) {
```

Comment:
Have some potentially naive questions here 😄 

* How are these github teams provisioned?  Is it with metaservice?  
* Have we evaluated using something like the internal acl/teams service?